app/src/main/java/ar/com/wolox/android/example/network/services/LoginService.kt

```diff
@@ -0,0 +1,13 @@
+package ar.com.wolox.android.example.network.services
+
+import ar.com.wolox.android.models.LoginBody
+import ar.com.wolox.android.models.User
+import retrofit2.Response
+import retrofit2.http.Body
+import retrofit2.http.POST
+
+interface LoginService {
+
+    @POST("https://w-android-traning-backup2.herokuapp.com/auth/sign_in")
```

Comment:
You can store the base URL inside the BaseConfiguration file. That way, here you could just use `@POST("/auth/sign_in")`